### ISSUE TITLE
Running in dockerized home assistant (ha core)

### ISSUE THREAD

**Issue by kettenbach-it** [johanson/irur#26]
Hi Rasmus,

I find your software very interesting and would like to use it.
However, I run my home assistant on a k8s cluster - so I don't have a supervisord and certainly no HA-OS.
I see that you have a Dockerfile in your repository - so I could easily build a container image.
As far as I can see your software consists of a server component and a vue.js frontend.
What I need to know: how does the integration into a HA core happen when the server is running?
Gruß
Volker

**Comment by johanson** [comment]
Hi, you're correct. You need a mqtt server that talks with your HA and the plugin in question.
You don't have to run it with a supervisor, just give both HA and the plugin correct credentials to talk with each other.

https://www.home-assistant.io/integrations/mqtt/

**Comment by kettenbach-it** [comment]
Yes, I get it. And connecting your software to MQTT is no problem at all!

The question is: how can your frontend be integrated into home assistant?
